msteams-platform/apps-in-teams-meetings/build-tabs-for-meeting.md

```diff
@@ -73,6 +73,27 @@ The following image shows a tab added to the meeting details page in the Teams m
 
    :::image type="content" source="../assets/images/side-panel-view.png" alt-text="Screenshot shows side panel view with the list of apps.":::
 
+#### Deep link to meeting side panel
+
+You can create a deep link to your tab app that can open in the meeting side panel. When users in a meeting select the deep link, it opens the meeting side panel in the meeting stage. If a user selects the deep link before or after the meeting, the deep link opens in a pre or a post-meeting tab respectively. 
+
+# [Meeting side panel](#tab/meeting-side-panel)
+
+   The following shows a deep link in meeting side panel:
+
+   :::image type="content" source="../assets/images/sidepanel-deeplink.gif" alt-text="Screenshot shows side panel view when deep link is selected.":::
+
+# [Pre or post-meeting tab](#tab/pre-or-post-meeting-tab)
```

Comment:
Please add some additional info on this. 

While it is true in a private meeting/group call/1:1 chat we can still navigate to the tab in chat pre/post meeting, we do not support this behavior in channel meetings.